### DISCUSSION TITLE
Proper way to add a new bundle config field?

### DISCUSSION THREAD

**Discussion by meowtec** [tauri-apps/tauri#4295] [Q&A] [answered]
I am trying to implement a new bundle config `fileAssociations` that electron has.
ref: #3736

I have modified some source files (WIP commit is here: https://github.com/meowtec/tauri/commit/71eebe924b15f36df50c1ec901b953039fc20ed8) and want to verify on my real project.

But when I try to build my app using the modified CLI, it got some error:

```log
thread 'main' panicked at 'error found during tauri-build: Error("unknown field `fileAssociations`, 
expected one of `active`, `targets`, `identifier`, `icon`, `resources`, `copyright`, `category`,
`shortDescription`, `longDescription`, `deb`, `macOS`, `externalBin`, `windows`", line: 1, column: 423)',
~/.cargo/registry/src/github.com-1ecc6299db9ec823/tauri-build-1.0.0-rc.12/src/lib.rs:165:5
```

Here is my Operation steps (the directory is `tauri`):

1. run `cargo build` at `tauri`
2. run `yarn && yarn build` at `tauri/tooling/cli/node`
3. run `node path/to/tauri/tooling/cli/node/tauri.js build` at my app project directory

In my app's tauri.conf.json:
```diff
      "category": "GraphicsAndDesign",
+     "fileAssociations": [
+      {
+         "ext": ["png", "jpg"]
+      }
      ],
```

Is there some step wrong? Or is there some file missing to be changed?

**Comment by meowtec** [comment, marked as answer]
Got it. I should also add `path = "path/to/local/tauri-build-and-tauri"` into my project's cargo.toml

**Comment by erguotou520** [comment]
Any update?
